project/app/assets/stylesheets/posts/index.scss

```
@@ -59,8 +61,37 @@
     width: 40em;
     height: 10em;
   }
+
+  .common {
+    padding: 0.25em 0.50em; 
+    border-radius: 0.25em / 0.25em; 
+    background: #39F; color: #FFF; 
```

@hiroty777 
[imo]
1行ずつに書いたほうが見やすいです

```suggestion
    background: #39F;
    color: #FFF; 
```